src/sid.cpp

```diff
@@ -1,5 +1,6 @@
 #include "sid.h"
 #include "util.hpp"
+#include "globals.h"
```

Comment:
don't do this. Using global variables negatively impacts code quality, because it makes it much more difficult to keep different components separate from each other using well-defined interfaces.

Instead, offer a `Sid::set_volume` function (which e.g. can be called by `loop.cpp` where all the other `Sid::set_whatever` functions are called) here.

(actually, try searching for `set_filter_mode` or any other of these setters and see where and how they are used. Use `set_volume` in a similar way)